aws-creds.py

```diff
@@ -74,8 +74,6 @@ def load_mfa_device():
     for device in mfa_out["MFADevices"]:
         # Only TOTP devices are supported
```

Comment:
Seems like this comment is probably stale? But it's not clear to me -- I guess we'd now accept e.g. `u2f` devices too. But those probably won't work with this script, since we will later request a TOTP code specifically?

Reply:
I'm not sure because I didn't test this.
We had a look at this script together with JD during my onboarding and we noticed there are many comments that aren't valid anymore. For example, now users can have more than one MFA device in AWS.

In this PR I just deleted the lines that block me from logging in, but certainly it would be nice to read this script entirely and re-verify all the assumptions 👍
Also, in the future we might want to use sso so maybe most of this script becomes not necessary? 🤔

Reply:
I'd like to see this particular comment dropped at least, but otherwise happy to merge and see if that breaks anything.